build-tools/mono-runtimes/mono-runtimes.targets

```diff
@@ -323,9 +317,27 @@
     />
     <Touch Files="@(_InstallMonoPosixHelperOutput);@(_InstallUnstrippedMonoPosixHelperOutput)" />
   </Target>
+  <Target Name="_GetMonodocItems" DependsOnTargets="_BuildRuntimes">
+    <ItemGroup>
+      <_MonoDocCopyItems Include="@(MonoDocCopyItem->'$(_MonoOutputDir)\%(Identity)')" />
+      <_MonoDocCopyItems
+          Condition=" Exists ('$(_MonoOutputDir)\%(Identity)') "
+          Include="@(MonoDocCopyItemOptional->'$(_MonoOutputDir)\%(Identity)')"
+      />
+    </ItemGroup>
+    <ItemGroup>
```

Comment:
We can collapse these into one ItemGroup. there is not need to separate them. unless @jonpryor said to :P